### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by Lihongjian520** [kcct-fujimotolab/3DCNN#19]
Can I use my own data set??? And how to use it????

**Comment by melaanya** [comment]
@Lihongjian520 Yes, I suppose you can, you should load the model from one of the provided json files (e.g.[ucf101cnnmodel.json](https://github.com/kcct-fujimotolab/3DCNN/blob/master/result_cnn_10class/ucf101cnnmodel.json)), with the function [`model_from_json`](https://keras.io/models/about-keras-models/) then modify it as you like, and further use it with your data.

**Comment by 1774467982** [comment]
> @Lihongjian520 Yes, I suppose you can, you should load the model from one of the provided json files (e.g.[ucf101cnnmodel.json](https://github.com/kcct-fujimotolab/3DCNN/blob/master/result_cnn_10class/ucf101cnnmodel.json)), with the function [`model_from_json`](https://keras.io/models/about-keras-models/) then modify it as you like, and further use it with your data.

Can you be more specific？
